repose-aggregator/tests/performance-tests/Jenkinsfile

```diff
@@ -27,14 +27,17 @@ stage("Performance Test") {
     def performanceJenkinsJob = "performance-test"
     def performanceTestParam = "perf_test"
     def extraOptionsParam = "extra_options"
+    def random = new Random()
 
     def jobsToRun = [:]
 
     for (int index = 0; index < perfTests.size(); index++) {
         def perfTest = perfTests[index]
         jobsToRun[perfTest] = {
             retry(3) {
-                build(job: performanceJenkinsJob, parameters: [
+                build(job: performanceJenkinsJob,
+                      quietPeriod: (random.nextInt() % 585) + 15,
```

Comment:
Can't `nextInt()` return a negative integer value? In which case, this could set the `quietPeriod` to a negative value. Not sure what the resultant behavior is, but it probably isn't what we want.